examples/firebase-auth-firestore/README.md

```diff
@@ -12,23 +12,34 @@ See the screen recording at `./screen_recording.gif` or Open this example on [Co
 
 ## Example
 
-To run it, you need to:
+To run it, you need to either:
 
-1. Create a Firebase Project
+### 1. Run against a Firebase Project
+
+1. [Create a Firebase Project](https://console.firebase.google.com)
 2. Enable Auth (with email) and Firestore
 3. Add a Web App
-4. Get the admin-sdk and client-sdk credentials
-5. Save them in the `.env`-file
+4. Get the [admin-sdk](https://firebase.google.com/docs/admin/setup#initialize-sdk) and [client-sdk credentials](https://firebase.google.com/docs/web/learn-more#config-object)
+5. Save them to SERVICE_ACCOUNT and CLIENT_CONFIG in the `.env`-file
+
+### 2. Use the Firebase emulators
```

Comment:
my personal preference would be to recommend the emulators as (1) but I'm biased 😄

Reply:
Let's keep it like this